### ISSUE TITLE
related to bypassing blocks - facets in search

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#252]
a user wants to perform a search by keyboard and use the facets, they want to access a particular facet item. currently keyboard access is at the facet label level, cannot tab into facet itself to access items

todo:
be able to tab into a facet item such as in "Type" 
be able to bypass out of tab facet without having to scroll through all items in facet
